zebra-chain/src/sapling/tree.rs

```diff
@@ -37,9 +37,13 @@ fn merkle_crh_sapling(layer: u8, left: [u8; 32], right: [u8; 32]) -> [u8; 32] {
     let mut s = bitvec![Lsb0, u8;];
 
     // Prefix: l = I2LEBSP_6(MerkleDepth^Sapling − 1 − layer)
-    s.extend_from_slice(&layer.bits::<Lsb0>()[0..6]);
-    s.extend_from_slice(&left.bits::<Lsb0>()[0..255]);
-    s.extend_from_slice(&right.bits::<Lsb0>()[0..255]);
+    s.extend_from_bitslice(&BitSlice::<Lsb0, _>::from_element(&layer)[0..6]);
+    s.extend_from_bitslice(
+        &BitSlice::<Lsb0, _>::from_slice(&left).expect("must work for small slices")[0..255],
+    );
+    s.extend_from_bitslice(
+        &BitSlice::<Lsb0, _>::from_slice(&right).expect("must work for small slices")[0..255],
+    );
```

Comment:
Similar to above, if you want to avoid the `expect`:

```suggestion
    s.extend_from_bitslice(&BitArray::<Lsb0, _>::from(left)[0..255]);
    s.extend_from_bitslice(&BitArray::<Lsb0, _>::from(right)[0..255]);
```